src/planner_test.cc

```diff
@@ -250,13 +274,24 @@ static void parametrizedAxisClamping(GCodeParserAxis defining_axis,
                                      GCodeParserAxis slowAxis) {
   const float kClampFeedrate = 10.0;
   const float kFastFactor = 17;  // Faster axis is faster by this much.
+  const float kRelativeError = 0.001;
```

Comment:
Maybe I'd call it `kAllowedRelativeError` so that it is more clear that we use that for testing.

Reply:
(solved)